src/patternfly/components/List/list.scss

```
@@ -8,20 +8,20 @@
   --pf-c-list--m-grid--GridGap: var(--pf-global--spacer--md);
 
   // Content variables (no modifier)
+  --pf-c-content--MarginBottom: var(--pf-global--spacer--md);
   --pf-c-content--li--MarginTop: var(--pf-global--spacer--sm);
   --pf-c-content--ul--PaddingLeft: var(--pf-global--spacer--lg);
-  --pf-c-content--ul--MarginTop: var(--pf-global--spacer--md);
-  --pf-c-content--MarginBottom: var(--pf-global--spacer--md);
-  --pf-c-content--ul--MarginLeft: var(--pf-global--spacer--lg);
+  --pf-c-content--ul--MarginTop: var(--pf-c-content--MarginBottom);
```

Why would we have `--pf-c-content--MarginBottom`? Couldn't we remove that and this line just be 

```suggestion
  --pf-c-content--ul--MarginTop: var(--pf-global--spacer--md);
```